test/integration/matchers/matcher-utils.ts

```diff
@@ -117,7 +117,7 @@ export function buildCellMatcher<R>(
 export function getFontStyleValue(expectedFont: ExpectedFont): number {
   return (
     (expectedFont
-      ? computeFontStyleValue(new Font(expectedFont.name, expectedFont.size, expectedFont.isBold, expectedFont.isItalic, expectedFont.isUnderline, expectedFont.isStrikeThrough))
+      ? computeFontStyleValue(new Font(expectedFont.family, expectedFont.size, expectedFont.isBold, expectedFont.isItalic, expectedFont.isUnderline, expectedFont.isStrikeThrough))
```

Comment:
**thought:** the parameter of the constructor of the `Font` object remains untouched.

As the `Font` object is part of the internal model, it's better to leave it untouched in this PR as it is currently propose.
Let's all keep in mind to change it later for consistency and clarity
`family` is a more appropriate naming as it matches the exact concept: the property is able to store more than a font name, a whole family i.e several font to use depending on the declaration order and the font availability.